### ISSUE TITLE
Xamarin forms -  upgrade iphone device to ios 16 - there is no sound any more from the previously running working app

### ISSUE THREAD

**Issue by pasha-o** [Azure-Samples/cognitive-services-speech-sdk#1661]
**Describe the bug**
Upgraded the iPhone to the latest ios 16 and suddenly there is no speech.

**To Reproduce**
1.  I can't debug the issue yet as the macOS team has difficulty running [xamarin forms for Xcode 14](https://github.com/xamarin/xamarin-macios/issues/15954) - @dalexsoto
2. You can recreate the issue by upgrading the ios to 16 and running a version of the xamarin forms app that was previously working with voice.

**Expected behavior**
the sound should be heard

**Version of the Cognitive Services Speech SDK**
Which version of the SDK are you using? 1.23.0

**Platform, Operating System, and Programming Language**
 - OS: ios 16
 - Hardware - iPhone, ARM
 - Programming language: C#, 

**Additional context**
- I can't get any log file  - this is as a result of simply updating the iPhone to the latest version of ios 16 and running the previously working app.

**Comment by ivanicin** [comment]
While I don't know if this is iOS 16 specific I can confirm that this sample doesn't produce any output:

```
static async Task SynthesizeAudioAsync()
{
    var config = SpeechConfig.FromSubscription("<paste-your-speech-key-here>", "<paste-your-speech-location/region-here>");
    using var synthesizer = new SpeechSynthesizer(config);
    await synthesizer.SpeakTextAsync("Synthesizing directly to speaker output.");
}
```

Also this has nothing to do with Xamarin.Forms as they are UI framework and as such has literary nothing with sound output, the proper title should refer to Xamarin.iOS.

**Comment by yulin-li** [comment]
Looks like this is a duplicated issue of #1613 . We have fixed this issue internally, which will be released with next version 1.24 around mid Oct.

**Comment by jhakulin** [comment]
Closing this as duplicate.
